### DISCUSSION TITLE
How to sync Page Properties between two instances?

### DISCUSSION THREAD

**Discussion by bsuchorowski** [requarks/wiki#6222] [Help / Questions] [answered]
Hi,

We have one instance of Wiki.Js that serves as a "Source of Truth" and has Git Storage enabled with "push" option. Another instance of Wiki.Js is set as sync replica from afornementioned git storage. Page content is syncing without issues. However, we have added a custom script to one of pages properies and it is not being synced. We thought that we could add this page property script manually on both environments - unfortunatelly, every later page content edit on source Wiki.Js clears Page Properties on a replica and we are back where we started.

Is there any good and working way how to sync Page Properties between wiki.js (v2) instances?

**Comment by NGPixel** [comment, marked as answer]
You need to use the high-availability mode:
https://docs.requarks.io/install/config#high-availability

Both instances must connect to the same PostgreSQL DB / DB cluster.

**Reply by bsuchorowski** [reply]
"We've been there". We rejected that approach as we wanted to have a common page content (Git Storage does the job, except the case described above) but separate server configuration (OpenID authentication to separate authentication servers).
